File: PolicyManagementAngular/src/app/app-entites/dtos/inspection/inspection-search-policy-dto.ts
export interface IInspectionSearchPolicyDto {
    ControlNumber: string;
    Customer: string;
    RegistrationNumber: string;
    Manufacturer: string;
    ManufacturerId: number;
    Model: string;
    ModelId: number;
    MakeYear: string;
    MakeYearId: number;
    InsuranceCompany: string;
    InsuranceCompanyId: number;
    Pos: string;
    PosId?: number;
    PolicyExpiryString: string;
    EngineNumber: string;
    ChassisNumber: string;
}

File: PolicyManagement.Api/PolicyManagement.Dtos/Inspection/InspectionSearchPolicyDto.cs
using System;

namespace PolicyManagement.Dtos.Inspection
{
    public class InspectionSearchPolicyDto
    {
        public string ControlNumber { get; set; }
        public string Customer { get; set; }
        public string RegistrationNumber { get; set; }
        public string Manufacturer { get; set; }
        public int ManufacturerId { get; set; }
        public string Model { get; set; }
        public int ModelId { get; set; }
        public string MakeYear { get; set; }
        public int MakeYearId { get; set; }
        public string InsuranceCompany { get; set; }
        public string EngineNumber { get; set; }
        public string ChassisNumber { get; set; }
        public int InsuranceCompanyId { get; set; }
        public string Pos { get; set; }
        public int? PosId { get; set; }
        public DateTime? PolicyExpiry { get; set; }
        public string PolicyExpiryString { get => PolicyExpiry.HasValue ? PolicyExpiry.Value.ToString("MM-dd-yyyy").Replace("-", "/") : string.Empty; }
    }
}
